plus-minus

```diff
@@ -0,0 +1 @@
+(plus-minus :repo "peterwu/plus-minus" :fetcher github :branch "main")
```

Comment:
Specifying branch shouldn't be necessary if that's your default branch (which MELPA prefers):
```suggestion
(plus-minus :repo "peterwu/plus-minus" :fetcher github)
```

Reply:
Hi. I've made necessary changes in terms of version number and package dependency (Emacs 25.1) in my github repo but would really like to keep the +/- prefix as I chose it specifically because it's short and meaningful for what this package intends to do. Thank you.